docs/guide/connecting.rst

```diff
@@ -32,8 +32,8 @@ name - just supply the uri as the :attr:`host` to
 ReplicaSets
 ===========
 
-MongoEngine supports :class:`~pymongo.mongo_replica_set_client.MongoReplicaSetClient`
-to use them please use a URI style connection and provide the `replicaSet` name in the
+MongoEngine supports replica sets through :class:`~pymongo.mongo_client.MongoClient`.
+To use them please use a URI style connection and provide the `replicaSet` name in the
 connection kwargs.
```

Comment:
Reading through [PyMongo docs](https://pymongo.readthedocs.io/en/4.2.0/api/pymongo/mongo_client.html?highlight=mongoclient#pymongo.mongo_client.MongoClient), I don't think it should be required to use a URI style connection. You should be able to pass a `replicaSet` kwarg into the `MongoClient` just fine.

However, I don't think the accuracy of this documentation is high enough of a priority right now, especially given that we're not using replica sets in this way anywhere. Just making a note for the future.